### PR TITLE
[Feature](load) Add submitter and comments to load job

### LINKED ISSUE #16876: [Feature] (load) Add submitter and comments to load job
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/doris/issues?q=is%3Aissue) and found no similar issues.

### Description

Submitter information and comment information are two very important functions in the data import task, and their importance to users is as follows:

1. Submitter information: Submitter information can help users understand the source and responsible person of the data. In the data import task, the cooperation between multiple users or teams may be involved, so knowing the source of the data can help users to better follow up and communicate issues. Additionally, if a problem occurs with the data, users can quickly contact the submitter for a solution.
2. Note information: Note information can help users better understand the meaning and purpose of the import task.

Submitter information and comment information are very important for users of data import tasks, which can help users better understand the source, meaning, and use of data, so as to better complete tasks.

### Use case

```
MySQL [demo]> show routine load \G
*************************** 1. row ***************************
                  Id: 15005
                Name: label_13_04_19_35_471968_2016386449
                User: root
          CreateTime: 2023-02-17 17:11:03
           PauseTime: NULL
             EndTime: NULL
              DbName: default_cluster:demo
           TableName: tc1b6_ne_load_job_control_test_resume_all_routine_job_tb
               State: RUNNING
      DataSourceType: KAFKA
      CurrentTaskNum: 1
       JobProperties: {}
    CustomProperties: {"group.id":"label_13_04_19_35_471968_2016386449_ff40264a-0fbb-4eaf-8c45-919cde738531"}
           Statistic: {"receivedBytes":0,"runningTxns":[4006],"errorRows":0,"committedTaskNum":1,"loadedRows":0,"loadRowsRate":0,"abortedTaskNum":0,"errorRowsAfterResumed":0,"totalRows":0,"unselectedRows":0,"receivedBytesRate":0,"taskExecuteTimeMs":10006}
            Progress: {"0":"55334"}
                 Lag: {"0":585}
ReasonOfStateChanged:
        ErrorLogUrls:
            OtherMsg:
             Comment: 1234567
1 row in set (0.002 sec)
```

```
MySQL [demo]> show load \G
*************************** 1. row ***************************
         JobId: 11001
         Label: label_06_04_07_06_651161_706414572
          User: root
         State: FINISHED
      Progress: ETL:100%; LOAD:100%
          Type: BROKER
       EtlInfo: unselected.rows=0; dpp.abnorm.ALL=0; dpp.norm.ALL=1009
      TaskInfo: cluster:N/A; timeout(s):14400; max_filter_ratio:0.01
      ErrorMsg: NULL
    CreateTime: 2023-02-17 13:48:19
  EtlStartTime: 2023-02-17 13:48:21
 EtlFinishTime: 2023-02-17 13:48:21
 LoadStartTime: 2023-02-17 13:48:21
LoadFinishTime: 2023-02-17 13:48:22
           URL: NULL
    JobDetails: {"Unfinished backends":{},"ScannedRows":0,"TaskNumber":0,"LoadBytes":0,"All backends":{},"FileNumber":0,"FileSize":0}
 TransactionId: 2
  ErrorTablets: {}
       Comment: 1234567
```

### Related issues

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by yangzhg**
# Proposed changes

Issue Number: close #16876

## Problem summary

Add submitter and comments to load job

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [x] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by yangzhg** [comment]
run feut

**Comment by yangzhg** [comment]
run p0

**Comment by yangzhg** [comment]
run buildall

**Comment by yangzhg** [comment]
run p0

**Comment by yangzhg** [comment]
run buildall

**Comment by yangzhg** [comment]
run allbuild

**Comment by yangzhg** [comment]
run buildall

**Comment by yangzhg** [comment]
run p0

**Comment by yangzhg** [comment]
run buildall

**Comment by yangzhg** [comment]
run p0

**Comment by yangzhg** [comment]
run p0

File: fe/fe-common/src/main/java/org/apache/doris/common/FeMetaVersion.java [hunk 1]
```diff
@@ -52,8 +52,11 @@ public final class FeMetaVersion {
     public static final int VERSION_115 = 115;
     // change Auto to rbac
     public static final int VERSION_116 = 116;
+    // add user and comment to load job
+    public static final int VERSION_117 = 117;
     // note: when increment meta version, should assign the latest version to VERSION_CURRENT
-    public static final int VERSION_CURRENT = VERSION_116;
+
```
Comment (cambyzju) [inline fe/fe-common/src/main/java/org/apache/doris/common/FeMetaVersion.java:58]:
```suggestion
```

**Review by xy720 — APPROVED** [review]
LGTM

**Pull request merged**